src/index.html

```diff
@@ -13,6 +13,144 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <section>
+      <div class="stars stars--0">
+        <span
+          class="stars__star"
+          aria-label="Inactive star"
+        ></span>
```

Comment:
The 'span' tag is being used here. While it's not incorrect, a more semantic tag could be used, like 'li' if you decide to change the parent 'div' to 'ul'. This would improve the semantics of your HTML and help with accessibility and SEO.